.github/workflows/main.yml

```diff
@@ -90,10 +90,22 @@ jobs:
       - name: install gocovmerge
         run: make gocovmerge
 
-      - name: CLI
+      - name: CLI mysql index
         run: ./tests/e2e-test.sh
-      - name: Refactor-e2e # this will a WIP to move all the tests to respective packages
+        env:
+          INDEX_BACKEND: mysql
+      - name: CLI redis index
+        run: ./tests/e2e-test.sh
+        env:
+          INDEX_BACKEND: redis
+      - name: Refactor-e2e mysql index # this will a WIP to move all the tests to respective packages
```

Comment:
It looks like this refactor was started two years ago and never finished, I'm not sure it's worth duplicating these steps or which step should be preferred.

Reply:
this seems reasonable to me at first glance